core/kazoo_documents-1.0.0/src/kz_documents.hrl

```diff
@@ -3,5 +3,7 @@
 -include_lib("whistle/include/wh_types.hrl").
 -include_lib("kazoo_documents/include/kazoo_documents.hrl").
 
+-define(DEFAULT_TIMEZONE, <<"America/Los_Angeles">>).
+
```

Comment:
@jamesaimonetti can we default this to whapps_account_config:get_global or whapps_config:get  ?